File: src/main/java/me/oriharel/machinery/inventory/InventoryFillmentItem.java
package me.oriharel.machinery.inventory;

import org.bukkit.Material;
import org.bukkit.inventory.ItemStack;

public class InventoryFillmentItem extends InventoryItem {
    public InventoryFillmentItem(Material material, int amount, String displayName, String... lore) {
        super(-1, material, amount, displayName, lore);
    }

    public InventoryFillmentItem(ItemStack itemStack) {
        super(-1, itemStack);
    }

    public InventoryFillmentItem(Material material, int amount, String displayName) {
        super(-1, material, amount, displayName);
    }
}

File: src/main/kotlin/me/oriharel/machinery/inventory/InventoryFillmentItem.kt
package me.oriharel.machinery.inventory

import org.bukkit.Material
import org.bukkit.inventory.ItemStack

class InventoryFillmentItem : InventoryItem {
    constructor(material: Material?, amount: Int, displayName: String?, vararg lore: String?) : super(-1, material, amount, displayName, *lore) {}
    constructor(itemStack: ItemStack?) : super(-1, itemStack)
    constructor(material: Material?, amount: Int, displayName: String?) : super(-1, material, amount, displayName) {}
}